<launch>
	<arg name="gui" default="True" />
	<param name="robot_description" command="$(find xacro)/xacro.py '$(find inmoov_description)/robots/inmoov.urdf.xacro'" />
	<param name="use_gui" value="$(arg gui)"/>
	<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
		<rosparam param="source_list">["joint_command"]</rosparam>
		<rosparam param="rate">5</rosparam>
	</node>
	<node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find inmoov_description)/urdf.rviz" required="true" />
</launch>
